### DISCUSSION TITLE
Templates

### DISCUSSION THREAD

**Discussion by LetsGoMaan** [ebullient/ttrpg-convert-cli#573] [Q&A]
Hi! Please advise about the template. I want to use the functionality of Fantasy statblock, but after the CLI process I want to translate all the notes into another language (since English is not my first language). The problem is that if I use monster2md-yamlStatblock-body.txt, the translator does not translate all the text in it. But if I use the standard monster2md.txt, everything is translated. Therefore, could you provide me with the structure of monster2md-yamlStatblock-body.txt in full format, where each field is described, instead of
```statblock
{resource.5eStatblockYaml}
```
So that I can change it and make it more like the standard template. The problem is mainly in the table with characteristics and a couple of other fields.

**Comment by ebullient** [comment]
I think you're misunderstanding the cause here.

`{resource.5eStatblockYaml}` is a blob because it takes all of the attributes otherwise available (and in use in the other templates), and uses Jackson to write out well-formed, properly wrapped and escaped YAML.

You already have access to the raw parts: see the values used in the markdown template.

In this case, my guess is that your translator ignores codeblocks, and so skips over all of the YAML content entirely. If you can teach it to ingest/process/re-emit the YAML, then you'd be in good shape.
